### PR TITLE
Op 755 billing dashbaord loader

### PR DESCRIPTION
**Pull request by tsognong**

### CODE REVIEW

File: src/components/accessories/billsRecap/BillsRecap.tsx [hunk 1]
```diff
@@ -323,70 +327,66 @@ export const BillsRecap: FC = () => {
...
+              options={getOptions(
+                t("bill.bestsellingbyquantity"),
+                true,
+                isMobile(width) ? 1 : 3,
+                year.value
+              )}
+              data={getDataFromObject(
+                "bestSellingByQuantity",
+                t("bill.totalquantity")
+              )}
```
Comment (axelpezzo) [inline src/components/accessories/billsRecap/BillsRecap.tsx:342]: I would use a ternary condition here in order to show or a CircularProgress or the Bar component.

Comment (tsognong) [reply src/components/accessories/billsRecap/BillsRecap.tsx:342]: May I missed your explanation on Jira.

Comment (tsognong) [reply src/components/accessories/billsRecap/BillsRecap.tsx:342]: But when we replace the chart div totally by the loader it does'nt look nice. That's why I just prefer to show the loader over the bar div. What do you think?

Comment (axelpezzo) [reply src/components/accessories/billsRecap/BillsRecap.tsx:342]: Yeah sounds fine and, actually, the result in the UI is good.

**Pull request merged**